developers/weaviate/client-libraries/typescript/v2_v3_migration.md

```diff
@@ -324,7 +324,7 @@ console.log(JSON.stringify(result, null, 2));
 
 ### Generate Namespace
 
-The v3 client adds a new namespace, `generate`. Use the generate namespace like the query namespace to make queries.
+The v3 client adds a new namespace, `generate` for generative queries. This makes it easier to distinguish generative queries and a base vector searches. 
```

Comment:
Typo:
and base vector searches.

Consider:
distinguish between generative queries and vector searches.